src/config.rs

```diff
@@ -1,32 +1,31 @@
 extern crate directories;
 
-use std::{path::PathBuf, fmt::Display, error::Error};
+use std::{error::Error, fmt::Display, path::PathBuf};
 
 use serde::{Deserialize, Serialize};
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct Config {
     pub api_key: String,
     pub server_url: String,
+    pub allow_invalid_certs: bool,
```

Comment:
The way this is implemented currently, I think it'll explode on any existing config file that doesn't have the `allow_invalid_certs` setting explicitly specified.

I think making this config setting optional (with a default of "false") might be a sound move. I poked at it some locally, and came up with the attached patch of diffs.

I can merge this PR, but then I'll apply those diffs before I cut a release.